frontend/app/src/views/setting/ManageStatusCode.vue

```diff
@@ -335,7 +337,16 @@ export default {
           </v-tooltip>
         </span>
         <span>
-          <v-tooltip location="bottom">
+          <v-tooltip
+            location="bottom"
+            v-if="
+              $permissions.canUserPerform(
+                'addNewStatus',
+                this.isAdmin,
+                this.isUser
```

Comment:
minor - the variable for better readability should be called isRegUser, is RegAdmin

Reply:
Sure will change this. Have to change it into the Store as well then.